src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -78,7 +80,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
 
-    Message message = new Message(user, text);
+    //replace user inputted text if it fits the defined patten into an img tag
+    String regex = "(https?://\\S+\\.(png|jpg))";
```

Comment:
I would consider expanding this to include `gif`, `bmp`, and `jpeg` (not the extra e).

Reply:
will do in the next iteration!